builds/jazz-build-module/aws-apigateway-module.groovy

```diff
@@ -75,4 +75,34 @@ def injectLambdaIntegration(method, filePath) {
   }
 }
 
+def getApigatewayInfo(stage, domain, service, accountDetails, key, config) {
```

Comment:
Can you move this function to utility-loader file and make respective changes in build-pack files?

Reply:
Since it is specific to api gateway info, it is better to keep it in the builds/jazz-build-module/aws-apigateway-module.groovy file itself.